code/game/objects/items/mail.dm

```diff
@@ -548,7 +548,7 @@
 	shady_mail.made_by_cached_name = user.mind.name
 
 	if(index == 1)
-		var/mail_name = tgui_input_text(user, "Enter mail title, or leave it blank", "Mail Counterfeiting")
+		var/mail_name = tgui_input_text(user, "Enter mail title, or leave it blank", "Mail Counterfeiting", max_length = 512) // sanitization handled by reject_bad_text, 512 is just here for UX
```

Comment:
Shouldn't this be capped at something like label or name limit (same for the `reject_bad_text` call)?